app/src/main/java/org/oppia/android/app/profile/AddProfileActivityPresenter.kt

```diff
@@ -65,8 +65,9 @@ class AddProfileActivityPresenter @Inject constructor(
       lifecycleOwner = activity
       viewModel = profileViewModel
     }
-    binding.addProfileActivityAllowDownloadSwitch.setOnCheckedChangeListener { _, isChecked ->
-      allowDownloadAccess = isChecked
+    binding.addProfileActivityAllowDownloadConstraintLayout.setOnClickListener {
```

Comment:
Will it be a good idea to make a text part clickable?

Reply:
Actually this has been confirmed we Chantel we need to make the 2 textviews + switch clickable. Mentioned in PR description too.